src/components/HomeSearchBox/index.tsx

```diff
@@ -6,9 +6,29 @@ const HomeSearchBox = () => {
   const { t } = useTranslation('home')
   const router = useRouter()
   const [searchValue, setSearchValue] = useState<string>('')
+  const [active, setActive] = useState(0)
+  const [filtered, setFiltered] = useState<string[]>([])
+  const [isShow, setIsShow] = useState(false)
 
   const handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    setSearchValue(event.target.value)
+    const suggestions : string[] = ['Alabama','Alaska','American Samoa', 'Bubble Holmes', 'Contra Hits', 'Amazon'];
+    const searchValue = event.currentTarget.value
+    const newFilteredSuggestions = suggestions.filter(
```

Comment:
Would be nice to have a `TODO:` comment here that this is a mock up and remove it when there's the backend.